pipeline_games/test_extract_games.py

```diff
@@ -1 +1,93 @@
-from extract_games import parse_app_id
+"""Script for testing extract_games functions"""
+import os
+import pytest
+from bs4 import BeautifulSoup
+
+from extract_games import get_html, parse_app_id_bs, parse_game_bs, parse_price_bs, system_requirements, get_genre_from_steam, get_developer_name, get_publisher_name, convert_to_csv
+
+
+def test_html_returns_a_string():
+    """Ensure that string is returned for a generic website"""
+    result = get_html("https://www.google.co.uk")
+
+    assert isinstance(result, str) is True
```

Comment:
I usually write: `assert isinstance(result, str) == True` but this looks much nicer!

Reply:
I think pylint like is True but happy to change to == if we're doing that everywhere